packages/frint-store/src/Store.js

```diff
@@ -68,12 +76,30 @@ function Store(options = {}) {
       this.exposedState$.next(state);
     });
 
-  this.dispatch({ type: '__FRINT_INIT__' });
-
   this.getState = this.getState.bind(this);
   this.dispatch = this.dispatch.bind(this);
-}
 
+  // for epic
+  this._input$ = null;
+  this._action$ = null;
+  this._epic$ = null;
+  this._epicSubscription = null;
+
+  if (this.options.epic) {
+    this._input$ = new Subject();
+    this._action$ = new ActionsObservable(this._input$);
+    this._epic$ = new Subject();
+
+    this._epicSubscription = this._epic$
+      .map(epic => epic(this._action$, this, this.options.thunkArgument))
+      .switchMap(output$ => output$)
```

Comment:
because the `.map()` above is returning an ActionObservable, and further chaining should be done once that one starts emitting values.

Instead of writing it as `.map().switchMap()`, we can actually write it as `.switchMap()` directly even.